tools/utils/src/path.ts

```diff
@@ -42,7 +42,8 @@ export class Path {
   }
 
   toFileUrl() {
-    return pathToFileURL(this.path);
+    const url = pathToFileURL(this.path).toString();
+    return url.startsWith('file://') ? url : `file://${url}`;
```

Comment:
wtf? toFileUrl will not always prefix the 'file://'?

Reply:
Two days ago I encountered an error on running `affine` commands that are lacking of `file://` prefix. I cannot reproduce the issue any longer 🤔

Reply:
I saw it happened on windows once and this function is meant to fix it. the problem must be somewhere else